src/security-council-mgmt/SecurityCouncilManager.sol

```diff
@@ -122,6 +159,45 @@ contract SecurityCouncilManager is
         for (uint256 i = 0; i < _securityCouncils.length; i++) {
             _addSecurityCouncil(_securityCouncils[i]);
         }
+
+        setMinRotationPeriodImpl(_minRotationPeriod);
+
+        // we do our own 712 functionality because inheriting the OZ version
+        // would change our storage layout
+        NAME_HASH = keccak256(bytes("SecurityCouncilManager"));
+        VERSION_HASH = keccak256(bytes("1"));
+    }
+
+    function postUpgradeInit(uint256 _minRotationPeriod, address minRotationPeriodSetter)
+        external
+    {
+        address proxyAdmin = ProxyUtil.getProxyAdmin();
+        require(msg.sender == proxyAdmin, "NOT_FROM_ADMIN");
```

Comment:
meh I think it is unnecessarily risky to not check for admin, but we can still check for clean storage
also it might better inline the ProxyUtil since it is used 1 time only

Reply:
updated to have both